config.rb

```diff
@@ -10,7 +10,7 @@
 activate :i18n
 
 # Layouts
-# https://middlemanapp.com/basics/layouts/
+page "/index.html", :layout => "landing"
```

Comment:
Also, should this go under Line 15 below, which also deals with per-page layout changes?